src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -123,7 +126,23 @@ function presentPendingRequest (transaction, annuled = false) {
   return presentRequest({ origin, event: event[2], stateDirection, status, type, eventTimestamp, counterpartyId, amount, notes, fees: fee })
 }
 
-function presentPendingOffer (transaction, annuled = false) {
+let counter = 0
+function presentPendingOffer (transaction, invoicedOffers = [], annuled = false) {
+  const invalidEvent = invoicedOffers.find(io => !io.Invoice)
+  if (invalidEvent) return new Error(`Error: invalidEvent found: ${invalidEvent}.`)
```

Comment:
It's being returned here so that is can be caught in the error filter (ie: `filter(tx => !(tx instanceof Error))`) for both the `allActionable` query (on line 287) and the `getPending` query (on line 343).  

The intent of this] approach is to ensure that if one transaction errors, the remainder of the transactions are still able to be returned and rendered in the UI.